view/frontend/templates/js/replacejs.phtml

```diff
@@ -264,6 +264,8 @@ if (!$block->isOnPageFromWhiteList() && !$block->isMinicartEnabled()) { return;
 
     };
 
+    var cart = {};
```

Comment:
Relying on a global scope variable with such a generic name can cause issues.
Is there a reason to get it out of the isolated function scope?
If no, please put it back in the function below.
If yes, rename to something more specific, boltCart for instance.

Reply:
Good point, thank you. Updated